Dockerfile_integration_tests

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8.5-slim-buster
```

Comment:
@biotom you can extend this file and overwrite the parameters:
Build the first one with a -t parameter, that gives a tag for that. Use that tag in the testing docker file: FROM tagged_name. And only add those that you want to redefine: CMD and WORKDIR.
https://stackoverflow.com/questions/36617904/extending-local-dockerfile

Reply:
Interesting, thanks!

Reply:
Are you happy for me to merge this and extend the dockerfile after?